Sources/ApolloCodegenLib/CodegenConfiguration/ApolloCodegenConfiguration+OperationManifestConfiguration.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+
+extension ApolloCodegenConfiguration {
+  
+  public struct OperationManifestConfiguration: Codable, Equatable {
+    
+    // MARK: - Properties
+    
+    /// Local path where the generated operation manifest file should be written.
+    public let path: String
+    /// The version format to use when generating the operation manifest. Defaults to `.persistedQueries`.
+    public let version: Version
+
+    public enum Version: String, Codable, Equatable {
+      /// Generates an operation manifest for use with persisted queries.
+      case persistedQueries
+      /// Generates an operation manifest for pre-registering operations with the legacy
+      /// [Automatic Persisted Queries (APQs)](https://www.apollographql.com/docs/apollo-server/performance/apq).
+      /// functionality of Apollo Server/Router.
```

Comment:
Need to update this documentation, as we discussed. (Re: legacy safelisting !== APQs)